kura/org.eclipse.kura.ai.triton.server/src/main/java/org/eclipse/kura/ai/triton/server/TritonServerLocalManager.java

```diff
@@ -119,7 +119,7 @@ private void stopScheduledExecutor() {
         }
     }
 
-    private boolean isLocalServerRunning() {
+    public boolean isLocalServerRunning() {
```

Comment:
Is this method used somewhere else? If no, please keep it private.

Reply:
I think the correct way to handle this would be to set it as below since we use it in `TritonServerServiceImpl`:

```suggestion
    protected boolean isLocalServerRunning() {
```

what do you think?